### DISCUSSION TITLE
GX sound effects

### DISCUSSION THREAD

**Discussion by KiraiEEE** [Godiesc/firefox-gx#103] [General]
i wounder if there is a way to add sound effects like in OperaGX

**Comment by Godiesc** [comment]
@KiraiEEE shared the add-on he/she found on Firefox add-ons store so I share for you all: [GX MODS](https://addons.mozilla.org/en-US/firefox/addon/gx-mods/)

![imagen](https://github.com/Godiesc/firefox-gx/assets/22057609/ce44df63-bf65-433f-869f-09c911cde394)

**Comment by KiraiEEE** [comment]
I found an addon called GX Mods for those interested
